### PR TITLE
Update fstg01 and fstg02 after `g_forest` refactor

### LINKED ISSUE #715: Updates corresponding to refactor of `g_forest`
Update implementation of `g_forest` following https://github.com/insightsengineering/tern/pull/1158

Merge after https://github.com/insightsengineering/tern/pull/1158 and once chevron release is complete.

### PR DESCRIPTION
**Pull request by edelarua**
Closes #715 

Merge after chevron release and https://github.com/insightsengineering/tern/pull/1158

### CODE REVIEW

File: R/fstg02.R [hunk 1]
```diff
@@ -74,9 +74,8 @@ fstg02_main <- function(adam_db,
   execute_with_args(
     g_forest,
     tbl = result,
-    ...,
-    gp = grid::gpar(fontsize = 7),
-    draw = FALSE
+    font_size = 7,
+    ...
```
Comment (clarkliming) [inline R/fstg02.R:78]:
```suggestion
    ...,
    font_size = 7
```

to allow font_size be updated through `...`

File: DESCRIPTION [hunk 2]
```diff
@@ -37,10 +37,10 @@ Imports:
     nestcolor (>= 0.1.1),
     purrr,
     rlang,
-    rlistings (>= 0.2.6),
-    rtables (>= 0.6.5),
+    rlistings (>= 0.2.7),
+    rtables (>= 0.6.6),
     stringr,
-    tern (>= 0.9.2),
+    tern (>= 0.9.3),
```
Comment (clarkliming) [inline DESCRIPTION:43]:
```suggestion
    tern (>= 0.9.3.9001),
```

**Comment by clarkliming** [comment]
thank you @edelarua . Let's wait until chevron released to CRAN and then merge this

**Comment by BFalquet** [comment]
test is failing because of a glitch in rlistings, see https://github.com/insightsengineering/rlistings/issues/195

**Review by BFalquet — APPROVED** [review]
I am approving it because the remaining issue has to be fixed in `rlistings`, not here.

**Pull request merged**